### PR TITLE
Drop support for non-ES requests

### PR DESCRIPTION
**Pull request by matthew-andrews**
@charypar  @i-like-robots  :boom:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,17 @@ var api = require('next-ft-api-client');
 ## `.content(opts)` - get V2 content
 - `uuid` - uuid (or array of uuids) of wanted content
 - `type` - type of content request (Article or Image, only used for debug purposes)
```
Comment (i-like-robots) [inline README.md]: No longer needed

Comment (matthew-andrews) [inline README.md]: f46b8d7 thanks!

**Comment by i-like-robots** [comment]
\0/

**Comment by matthew-andrews** [comment]
This is probably blocked on article page using the new index exclusively, right?

**Comment by i-like-robots** [comment]
> Matt Andrews [10:11 AM] 
otherwise images will break?

> Matt Hinchliffe [10:11 AM] 
Ahhh

> Matt Hinchliffe [10:12 AM]
Good point, maybe merge it into a beta branch and tag it as such then

> Matt Andrews [10:13 AM] 
nah either major version bump or it can wait a few days i dont mind

> Matt Hinchliffe [10:13 AM] 
OK let's leave it a few days

**Comment by charypar** [comment]
Yay! Another reason to do Financial-Times/next-graphql-api#26

**Comment by matthew-andrews** [comment]
`simplicity++`

**Pull request merged**